packages/expo-image/src/Image.types.ts

```diff
@@ -51,6 +51,12 @@ export type ImageSource = {
    * @platform web
    */
   webMaxViewportWidth?: number;
+  /**
+   * Whether the image is animated (i.e. an animated GIF or WEBP).
```

Comment:
```suggestion
   * Whether the image is animated (an animated GIF or WebP).
```

- Let's avoid using [latin abbreviations as per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations)
- For consistency across the doc, let's use `WebP` as this is how we refer it:

![CleanShot 2023-10-31 at 12 23 10@2x](https://github.com/expo/expo/assets/10234615/09929f11-64b3-4df6-9e50-8b0a2176a865)